### PR TITLE
feat: configuration tidy-up [TIKI-112]

### PR DESCRIPTION
**Pull request by srlk**
moving clusterName and routes under config

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -141,23 +141,21 @@ func Read(configFile string) (*Config, error) {
...
-		},
-	}
-
-	if err := yaml.Unmarshal(b, c); err != nil {
-		return nil, fmt.Errorf("could not unmarshal config file: %w", err)
+	c.RestConfig = restCfg
+	c.Egress = &Egress{
+		HTTPClientTimeout:       metav1.Duration{Duration: HTTPClientDefaultTimeout},
+		SnykAPIBaseURL:          SnykAPIDefaultBaseURL,
+		SnykServiceAccountToken: os.Getenv("SNYK_SERVICE_ACCOUNT_TOKEN"),
```
Comment (tommyknows) [inline internal/config/config.go:158]: I'm not quite sure why you changed this, as far as I can tell now user's Egress setting will be overwritten by the `c.Egress = &Egress{...}` part.

I think it should be exactly as it was before? 🤔 Create a "default" values & then unmarshal into it.

**Review by tommyknows — CHANGES_REQUESTED** [review]

File: internal/config/config.go (hunk 1 above)
Comment (srlk) [reply internal/config/config.go:158]: > Create a "default" values & then unmarshal into it.

Yeah, rolling that now. 👍

File: internal/config/config.go [hunk 2]
```diff
@@ -19,6 +19,8 @@ import (
 	"fmt"
 	"net/url"
 	"os"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/yaml"
```
Comment (tommyknows) [inline internal/config/config.go:23]: This is a bit weird :)

File: internal/config/helm_test.go [hunk 3]
```diff
@@ -40,14 +40,23 @@ func TestHelmChartConfig(t *testing.T) {
...
+			"clusterName": "default",
+			"routes": []map[string]interface{}{
+				{
+					"organizationID":         "umbrella-corp",
+					"namespaces":             []string{"*"},
+					"clusterScopedResources": true,
+				},
+			},
+		},
+		"config.routes": []map[string]interface{}{
```
Comment (tommyknows) [inline internal/config/helm_test.go:53]: 🤔

**Review by tommyknows — APPROVED** [review]
I'll approve this anyway, but I think both of my comments should be fixed before that :)

**Pull request merged**